.github/workflows/action.yml

```diff
@@ -0,0 +1,47 @@
+name: Github action to create EKS cluster using eksctl and install DOP on it
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install eksctl
+      run: |
+        curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+        sudo mv /tmp/eksctl /usr/local/bin
+        sleep 10
+        eksctl version
+
+    - name: Checkout code
+      uses: actions/checkout@master
+
+    - name: Run Shellcheck
+      uses: bewuethr/shellcheck-action@v2
+
+##     TODO:: There is no go-file in repo. Thatswhy we are
+##     commenting this lines for now. 
+#
+#     - name: Run golangci-lint
+#       uses: actions-contrib/golangci-lint@v1
+
+    - name: Configure AWS credentials from secrets
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+    
+    - name: Create EKS cluster using eksctl and Install kubera on cluster
```

Comment:
Can we break this into multiple actions i.e. scripts
one to install EKS
other one to install DOP on Kubernetes

Reply:
Yes we can do that but install EKS has only 3 commands @AmitKumarDas . Is it fine to create separate file for that

Reply:
Yes. Lets separate them.